misc/python/materialize/mzcompose/test_result.py

```diff
@@ -125,6 +125,4 @@ def extract_error_chunks_from_output(output: str) -> list[str]:
     error_output = output[: output.index("+++ !!! Error Report") - 1]
     error_chunks = error_output.split("^^^ +++")
 
-    # The first part is uninteresting, contains only general testdrive
-    # executable output, not what actually failed, so cut it off.
-    return [chunk.strip() for chunk in error_chunks if len(chunk.strip()) > 0][1:]
+    return [chunk.strip() for chunk in error_chunks if len(chunk.strip()) > 0]
```

Comment:
This is the revert of a previous PR, is this intended?

Reply:
I'm not 100% sure why it's sometimes not working in CI. Locally this worked fine. Better to have the useless information in some cases until we can figure that out.